parsers/src/main/java/vct/parsers/transform/PVLToCol.scala

```diff
@@ -241,13 +252,15 @@ case class PVLToCol[G](override val originProvider: OriginProvider, override val
     case Unit2(_) => Null()
     case Unit3(n) => const(BigInt(n))
     case Unit4(n) => FloatValue(BigDecimal(n), PVL.float64)
-    case Unit5(n) => FloatValue(BigDecimal(n.init /* take off final "f" */), PVL.float32)
-    case Unit6(_, inner, _) => convert(inner)
-    case Unit7(id, None) => local(id, convert(id))
-    case Unit7(id, Some(Call0(typeArgs, args, given, yields))) =>
+    case Unit5(s"${n}f") /* take off final "f" */ => FloatValue(BigDecimal(n), PVL.float32)
+    case Unit6(data) => StringValue(data.substring(1, data.length - 1))
```

Comment:
Do you mean the use of "substring"? Since pattern matching on `"` looks a bit messy I  went with substring, but I'm fine with any other way of writing this down. Or is there something else the matter?

Reply:
No, I mean in terms of backslash encoding and so on :)
I guess in PVL it doesn't matter much.